runtime/bindings/python/CMakeLists.txt

```diff
@@ -83,6 +85,16 @@ target_link_libraries(iree_runtime_bindings_python_PyExtRt
   iree::hal
   iree::hal::drivers
   iree::hal::utils::allocators
+  iree::base::internal::file_io
+  iree::base::internal::path
+  iree::io::file_handle
+  iree::io::formats::gguf
+  iree::io::formats::safetensors
```

Comment:
If we start directing more users to this, we/they should be careful about security: https://github.com/openxla/iree/blob/5b0fca4269bb2f23c23889ab6163f779694da55f/runtime/src/iree/io/formats/safetensors/safetensors_format.h#L26-L33

In the Python bindings we could use an implementation that takes heavier (and hopefully safer / more completely tested) dependencies than the C impl that the base IREE tools can use.

Reply:
The path to that is to more thoroughly test this stuff and have some bridges to the outside. I'll leave it now.